setup.py

```diff
@@ -1,10 +1,11 @@
 from setuptools import setup
 
 install_requires = [
-    "huggingface_hub",
+    "huggingface_hub==0.8.1",
```

Comment:
Awesome work thanks @osanseviero 🤗 that will help to reduce the friction in usage.

Just a quick question why== and not >= or we want to keep this version to avoid bugs in the future?

Reply:
Thanks 🤗 we usually try to pin a version to prevent backwards compatibility issues, but you're right, we could have >= as well! Up to you :)

Reply:
✅ for now let's keep == this way we don't have any bugs if there is a major change in huggingface_hub.